en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md

```diff
@@ -8,19 +8,39 @@ The user has only the `apim:api_view` permission.
 Let's create a read only user and experience the Publisher portal in read only mode.
 
 Follow the instructions below to create a Read only user:
-1. Create role (e.g., `read_only`) in admin console.
-2. Create a user and assign the read_only role to the user.
-3. Assign the login permission to login to the Publisher portal via the admin console.
-4. Log in to the Admin Portal and navigate to **scope management** which is under **Settings**.
-5. Click **Add Scope Mappings**.
+
+1. Create role (e.g., `read_only`) in admin console.Sign in to the management console (`https://<APIM_Host>:<APIM_Port>/carbon`) 
+   as the admin (default credentials are admin/admin). 
+   
+2. Click **Main**, and then click **Add** under **Users and Roles**.
+        
+      [![Add users and roles]({{base_path}}/assets/img/administer/add-users-and-roles.png)]({{base_path}}/assets/img/administer/add-users-and-roles.png)
+
+3. Click **Add New Role**.
+
+      ![Add new role]({{base_path}}/assets/img/administer/add-new-role.png)
+
+4.  Enter the name of the user role (e.g., `read_only`).
+
+      ![read-only-user-creation]({base_path}}/assets/img/learn/api-security/read_only_role.png)
+
+5. Create a user and assign the read_only role to the user.
+   
+6. Assign the login permission to login to the Publisher portal via the admin console.
+   
+7. Log in to the Admin Portal and navigate to **scope management** which is under **Settings**.
+   
+8. Click **Add Scope Mappings**.
    Enter the Role name as `read_only` (This is the role you assigned to the read only user).
```

Comment:
This will not render as a new line as you have not added a space before this line.

Reply:
Indentation of line 34 is wrong. This indentation will break the numbering.